tendenci/apps/forms_builder/forms/urls.py

```diff
@@ -17,6 +17,11 @@
     re_path(r'^%s/entries/export/(?P<task_id>[-\w]+)/check/$' % urlpath, views.entries_export_check, name="form_entries_export_check"),
     re_path(r'^%s/entries/export/(?P<task_id>[-\w]+)/download/$' % urlpath, views.entries_export_download, name="form_entries_export_download"),
 
+    re_path(r"^%s/entry/edit/(?P<id>\d+)$" % urlpath, views.form_detail, name="form_entry_edit"),
```

Comment:
I don't see `views.form_detail` handles entry edit. Besides, it accepts "slug",  not "id", as the parameter. Did you forget to implement it?